process/block/baseProcess_test.go

```diff
@@ -89,7 +89,7 @@ func createShardedDataChacherNotifier(
 }
 
 func initDataPool(testHash []byte) *mock.PoolsHolderStub {
-	rewardTx := &rewardTx.RewardTx{
+	rwdTx := &rewardTx.RewardTx{
```

Comment:
you should leave it as rewardTx as it is clearer

Reply:
It is a conflict with the imported modules.